pyproject.toml

```diff
@@ -54,16 +54,16 @@ filterwarnings = [
     "default:Error (reading|writing) persistent compilation cache entry for 'jit_equal'",
     "default:Error (reading|writing) persistent compilation cache entry for 'jit__lambda_'",
     "default:jax.extend.mlir.dialects.mhlo is deprecated.*:DeprecationWarning",
-    "default:jax.xla_computation is deprecated. Please use the AOT APIs.*:DeprecationWarning",
+    
     # TODO(jakevdp): remove when array_api_tests stabilize
-    # start array_api_tests-related warnings
     "default:.*not machine-readable.*:UserWarning",
     "default:Special cases found for .* but none were parsed.*:UserWarning",
     "default:.*is not JSON-serializable. Using the repr instead.",
-    # end array_api_tests-related warnings
-    # This is a transitive warning coming from TensorFlow dependencies.
+
+    # These are transitive warnings coming from TensorFlow dependencies.
     # TODO(slebedev): Remove once we bump the minimum TensorFlow version.
     "default:The key path API is deprecated .*",
+    "default:jax.xla_computation is deprecated.*:DeprecationWarning",
```

Comment:
I'm only leaving this here because `import tensorflow` triggers this in tf 2.17.0, and there's no more fine-grained way to cause pytest to ignore this warning when collecting `jax/experimental/jax2tf/jax2tf_test.py`